update.sh

```diff
@@ -17,13 +17,16 @@ print_header() {
 	#                       PLEASE DO NOT EDIT IT DIRECTLY.
 	# ------------------------------------------------------------------------------
 	#
-	EOI
+EOI
```

Comment:
Why change the indentation of lines 11 and 20? The other methods use the same kind of indentation.

Reply:
Fixed